Download all translations at once

markoheijnen opened the pull request:
The list table logic is missing.

markoheijnen committed:
Have a seperate GlotPress file for connections :)

markoheijnen committed:
Make GlotPress_API extendable for other usage

markoheijnen committed:
Fixes caching result instead of dataset in get_locale_data()

markoheijnen committed:
Moving more code to GlotPress_API class.

markoheijnen committed:
I love tabs. Also removed closing PHP tag

markoheijnen committed:
First basic WP_List_Table

markoheijnen commented:
The use case of it would be http://wp-stats.nl. It's a static website that shows some WordPress statistics. Most of the things in english are day or week names. So I can use that on the site.

stas commented:
Looking forward.

I have a bunch of patches in the queue for GlotPress (like #1 fixes and so on). Maybe I can make an effort and merge those by the end of this week.

markoheijnen committed:
Caching differently. Cache on the functions that calls self::fetch

markoheijnen committed:
Removed all caching from localize.php

markoheijnen committed:
Small update to list table

Still need to find a way to get the real locale

markoheijnen committed:
Change names of bulk actions

markoheijnen committed:
Add sort to list table

markoheijnen committed:
Add help information to help tab

markoheijnen committed:
Remove all not needed functionality

markoheijnen committed:
Add locale function for transforming the data from GlotPress

markoheijnen committed:
Add support for multiple GlotPress projects

Download .mo files for default WordPress translation and be able to set WPLANG as a (super) admin.

markoheijnen committed:
Download GlotPress file on setting the default language
